app_server.py

```diff
@@ -49,9 +49,9 @@ def cdn_url_for(*args, **kwargs):
 
 def load_lending_policy(policy):
     if not policy:
-        logging.info("No lending policy.")
         return {}
     if isinstance(policy, basestring):
+        logging.info("Lending policy: %s" % policy)
```

Comment:
This is an unrelated change that's been bugging me for a long time. Not having a lending policy isn't interesting; it's having one that's interesting.

Reply:
I put this into a separate branch since it's totally unrelated.